.github/workflows/build_and_test_debug.yml

```diff
@@ -0,0 +1,183 @@
+name: Build and Test
+
+concurrency:
+  group: ${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
```

Comment:
strangely, `ubuntu-20.04` can't seem to find any runners. This works in the client repository, however.